crates/autopilot/src/database/auction.rs

```diff
@@ -72,7 +60,7 @@ impl QuoteStoring for Postgres {
 }
 
 impl Postgres {
-    pub async fn solvable_orders(&self, min_valid_to: u32) -> Result<SolvableOrders> {
+    pub async fn solvable_orders(&self, min_valid_to: u32) -> Result<domain::SolvableOrders> {
```

Comment:
`boundary::SolvableOrders`

Reply:
IIRC using `domain::SolvableOrder` would be fine here. The `DB` is an infra piece so it should not be the concern of the `domain` logic to cater to the needs of the `infra` but rather the `infra` should take whatever the `domain` gives it and bend over backwards to make it work in order to have the `domain` logic be as clean as it can be.
So when the remaining `database` code gets DDD-ified down the line it should already have the expected type.
Am I missing something?